src/redux/store.js

```diff
@@ -1,9 +1,13 @@
-import { createStore } from "redux"
-import todoLists from "./reducers/todoLists"
+import { createStore } from "redux";
+import todoLists from "./reducers/todoLists";
 
-const initialState = JSON.parse(localStorage.getItem("dataState"))
+const initialState = JSON.parse(localStorage.getItem("dataState"));
 console.log(initialState);
 
-const store = createStore(todoLists, initialState)
+const store = createStore(
+  todoLists,
+  initialState,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
That is the only real change, the rest is my atom editor auto save with prettify feature